docs/reference/commandline/ps.md

```diff
@@ -47,7 +47,6 @@ Running `docker ps --no-trunc` showing 2 linked containers.
 
 ```bash
 $ docker ps
-
```

Comment:
Can you add these lines back? These were added because it was difficult to tell where the _command_ to run stopped, and the _output_ (response) of the command began; adding a blank line makes that slightly more readable (even though it doesn't "match" the actual output)